tests/tpchbench/src/main/scala/sparkcyclone/tpch/TPCHBenchmark.scala

```diff
@@ -270,8 +270,12 @@ object TPCHBenchmark extends SparkSessionWrapper {
     }
 
     queries.foreach { case (query, i) =>
-      if (!toSkip.contains(i)) {
+      try {
```

Comment:
I'd say do this behavior only when there is more than 1 query to run. I'd want in the new runner to capture failed runs programmatically rather than by looking.